src/main/java/seedu/address/logic/parser/DeleteCandidateCommandParser.java

```diff
@@ -21,8 +21,8 @@ public DeleteCandidateCommand parse(String args) throws ParseException {
             Index index = ParserUtil.parseIndex(args);
             return new DeleteCandidateCommand(index);
         } catch (ParseException pe) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCandidateCommand.MESSAGE_USAGE), pe);
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, pe.getMessage())
+                            + "\n" + DeleteCandidateCommand.MESSAGE_USAGE, pe);
```

Comment:
I just played around with the PR.

![image](https://user-images.githubusercontent.com/61644924/140299251-0338a18a-574b-4875-b971-a0c952f07fbf.png)

Should the Index error really be shown when no index is inputted

Reply:
On the first look, the reason for this behaviour seems to be that the parsed index is an empty string (""). I feel that this case can be handled similarly to other non-numerical user inputs. Let me know what are your thoughts.

Reply:
We had a discussion on this in the tele group